Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -327,7 +354,7 @@ - (void)updateCall:(OCTCall *)call withState:(OCTToxAVCallState)state pausedStat
 
 - (void)checkForCurrentActiveCallAndPause
 {
-    if (! [self.audioEngine isAudioRunning:nil]) {
+    if (! [self.audioEngine isAudioRunning:nil] && ! [self.videoEngine isVideoSessionRunning]) {
```

Comment:
Hm, if video is disabled then
- `audioEngine` would be running
- `videoEngine` wouldn't be running
- call wouldn't be put on pause because of `video`